src/lib/viewers/BaseViewer.js

```diff
@@ -569,6 +570,9 @@ class BaseViewer extends EventEmitter {
             this.annotator.emit(ANNOTATOR_EVENT.setVisibility, false);
             this.disableAnnotationControls();
         }
+        if (this.fullscreenToggleEl && this.fullscreenToggleEl.focus) {
```

Comment:
Does focus on fullscreen exit work correctly?

Reply:
@hormaza, did we check this case? What was the finding?